srcpkgs/gdm/template

```diff
@@ -11,23 +11,24 @@ configure_args="
  -Dsystemd-journal=false -Dinitial-vt=7 -Dwayland-support=true
  -Dselinux=disabled -Dlibaudit=disabled -Dgdm-xsession=true
  -Ddefault-path=/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
- -Dsystemdsystemunitdir=/usr/lib/systemd/system
- -Dsystemduserunitdir=/usr/lib/systemd/user"
+ -Dsystemdsystemunitdir=no
+ -Dsystemduserunitdir=no
```

Comment:
Import the following patch and fix the paths.
https://gist.github.com/oreo639/bde1b7b04a7f1a5d8b1fbc525429ac4c

(That patch has been submitted upstream btw: https://gitlab.gnome.org/GNOME/gdm/-/merge_requests/247)

Reply:
fixed